scripts/ci/qemu-run.sh

```diff
@@ -129,10 +134,6 @@ cleanup() {
 
 trap cleanup INT
 
```

Comment:
```suggestion
parse_args() {
  while [[ $# -gt 0 ]]; do
    case $1 in
      -v|--verbose)
        set -x
        shift
        ;;
      -h|--help)
        usage
        exit 0
        ;;
      -*)
        usage
        echo "Unknown option $1"
        exit 1
        ;;
      *)
        POSITIONAL_ARGS+=( "$1" )
        shift
        ;;
    esac
  done
}

parse_args "$@"
set -- "${POSITIONAL_ARGS[@]}"
```

That way:

- You don't need to remove next if
- You can use arguments anywhere

If you apply then bring back 

```sh
if [ $# -ne 2 ]; then
  usage
fi
```

And add `--help` and `--verbose` to usage.
I mostly use this template: https://github.com/m-iwanicki/scripts/blob/main/scripts/script-template.sh when writing scripts.

Reply:
You can leave an issue about style.

Reply:
Ok, I will fix this.

Reply:
05f768645361ec3dc19adc23bbee153618c46f8f

Reply:
05f768645361ec3dc19adc23bbee153618c46f8f..830ee65fcbc61b7756de28f35a7da6d88bc411bf